vespajlib/src/test/java/com/yahoo/tensor/TensorAddressTestCase.java

```diff
@@ -79,5 +79,19 @@ void testPartialCopy() {
         int[] o_1_3_2 = {1,3,2};
         equal(ofLabels("b", "d", "c"), abcd.partialCopy(o_1_3_2));
     }
+    
+    @Test
+    void testHashCodeForLowEntropy() {
+        var e = TensorAddress.ofLabels("1", "4", "5", "6", "x", "y", "z");
+        var f = TensorAddress.ofLabels("1", "4", "5", "6", "x", "y", "z");
+        assertEquals(e.hashCode(), f.hashCode());
+        
+        var a = TensorAddress.ofLabels("a", "b", "c", "d", "e", "f", "g");
+        var b = TensorAddress.ofLabels("a", "b", "c", "d", "e", "f", "g", "h", "i", "j");
+        assertNotEquals(a.hashCode(), b.hashCode());
 
+        var c = TensorAddress.ofLabels("1", "4", "5", "6", "x", "y", "z");
+        var d = TensorAddress.ofLabels("1", "3", "5", "7", "z", "b", "c", "d", "e", "f");
+        assertNotEquals(c.hashCode(), d.hashCode());
```

Comment:
The values tested here are sufficiently esoteric (as they were chosen to produce collisions with the old hash functions) that I think it would be useful with a comment explaining why they were included.

Reply:
Agree, added a comment.